extension/persistence/eclipselink/src/main/java/org/apache/polaris/extension/persistence/impl/eclipselink/PolarisEclipseLinkMetaStoreSessionImpl.java

```diff
@@ -701,9 +701,9 @@ public int lookupEntityGrantRecordsVersion(
             principalSecrets.getPrincipalId());
 
     // rotate the secrets
-    principalSecrets.rotateSecrets(mainSecretToRotate);
+    principalSecrets.rotateSecrets();
     if (reset) {
-      principalSecrets.rotateSecrets(principalSecrets.getMainSecret());
```

Comment:
Hmm, the reason we passed in the original secret was for idempotence. It's imperative to make sure that if a rotate request is sent twice accidentally (e.g., client retry), the keys aren't rotated twice

Reply:
That is a good idea, we should probably document that on the method if it isn't already